darwin/exporter/formats/coco.py

```diff
@@ -401,7 +82,7 @@ def _calculate_tag_categories(
                 categories[annotation_class.name] = _calculate_category_id(
                     annotation_class
                 )
-    return categories
+    return dict(sorted(categories.items(), key=lambda item: item[1]))
```

Comment:
These changes are to Ensure COCO categories are always in the same, ascending order to avoid E2E flakiness